frontend/src/app/main/data/viewer.cljs

```diff
@@ -196,32 +204,39 @@
     ptk/UpdateEvent
     (update [_ state]
       (let [increase #(min (* % 1.3) 200)]
-        (update-in state [:viewer-local :zoom] (fnil increase 1))))))
+        (-> state
+            (update-in state [:viewer-local :zoom] (fnil increase 1))
+            (assoc-in  [:viewer-local :zoom-type] :default))))))
```

Comment:
I'm not clearly understand the need to assign the zoom-type here and in the following events.

Reply:
In fact, looking the code, the zoom-type prop is not used in any view, so I think we can just :fire: it